src/field_warp_tasks.c

```diff
@@ -195,16 +195,16 @@ void sub_02053038(FieldSystem *fsys, BOOL isConnection) {
     if (weather == 11 && SysFlagFlashCheck(scriptState) == TRUE) {
         weather = 12;
     }
-    FlyPoints_SetWeatherType(flyPoints, weather);
+    LocalFieldData_SetWeatherType(localFieldData, weather);
     if (isConnection) {
-        GF_ASSERT(FlyPoints_GetCameraType(flyPoints) == MapHeader_GetCameraType(mapId));
+        GF_ASSERT(LocalFieldData_GetCameraType(localFieldData) == MapHeader_GetCameraType(mapId));
     } else {
-        FlyPoints_SetCameraType(flyPoints, MapHeader_GetCameraType(mapId));
+        LocalFieldData_SetCameraType(localFieldData, MapHeader_GetCameraType(mapId));
     }
     if (!isConnection) {
         spawnId = MapHeader_GetSpawnIdForDeathWarp(mapId);
         if (spawnId != 0) {
-            FlyPoints_SetDeathSpawn(flyPoints, spawnId);
+            LocalFieldData_SetDeathSpawn(localFieldData, spawnId);
```

Comment:
We don't really use the term 'death' in pokemon games, maybe SetRespawnPoint or SetBlackoutSpawn?